cmd/internal/shared/console/config.go

```diff
@@ -43,6 +47,7 @@ var DefaultConsoleConfig = console.Config{
 			JSFiles:       []string{"console.js"},
 		},
 		FrontendConfig: console.FrontendConfig{
+			DocumentationBaseURL: fmt.Sprintf("https://thethingsstack.io/%s", strings.TrimSuffix(strings.TrimPrefix(version.TTN, "v"), "-dev")),
```

Comment:
I thought we were getting rid of publishing versioned docs? cc @htdvisser

Reply:
1. Documentation links in the console should point to the exact version of the console that the user is currently working with.
2. At the moment this PR is submitted we still have versioned docs.